lib/tasks/db.rake

```diff
@@ -1,6 +1,22 @@
 # frozen_string_literal: true
 
+# We are providing our own task with our own format
+Rake::Task['db:encryption:init'].clear
```

Comment:
It feels sort of odd to replace the framework task instead of adding a new task?

Reply:
I guess we could add another task with another name, but this one serves the same purpose. I'm open to suggestions though.